src/Atata/Attributes/ControlSearch/FindAttribute.cs

```diff
@@ -85,7 +85,7 @@ public ScopeSource ScopeSource
 
         /// <summary>
         /// Gets or sets the outer XPath.
-        /// The default value is null, meaning that the control is searchable as descendant (using ".//" XPath) in scope source.
+        /// The default value is <see langword="null"/>, meaning that the control is searchable as descendant (using <c>.//</c> XPath) in scope source.

```

Comment:
Wrap XPath values also with the double quotes.

```suggestion
        /// The default value is <see langword="null"/>, meaning that the control is searchable as descendant (using <c>".//"</c> XPath) in scope source.
```